lib/teiserver/microblog/libs/post_lib.ex

```diff
@@ -197,6 +213,9 @@ defmodule Teiserver.Microblog.PostLib do
     query = "DELETE FROM microblog_post_tags WHERE post_id = $1;"
     {:ok, _} = Ecto.Adapters.SQL.query(Repo, query, [post.id])
 
+    query = "DELETE FROM microblog_poll_responses WHERE post_id = $1;"
+    {:ok, _} = Ecto.Adapters.SQL.query(Repo, query, [post.id])
```

Comment:
Slightly out of scope, but since you're touching this function, it would be nice to make all the deletion transactional, to avoid deleting half of the stuff if a later error comes.

Reply:
Good idea, updating.